modules/Campaigns/WizardNewsletter.html

```diff
@@ -104,6 +133,8 @@
 {$DIV_JAVASCRIPT}
 {$JAVASCRIPT}
 <script language="javascript">
-{$HILITE_ALL}
+  {
+    $HILITE_ALL
+  }
```

Comment:
Hi @Dillon-Brown, here we probably should also change to `{$HILITE_ALL}`

Reply:
Pushed!